pkg/skaffold/initializer/init.go

```diff
@@ -254,11 +254,50 @@ func processCliArtifacts(artifacts []string) ([]builderImagePair, error) {
 			return nil, fmt.Errorf("malformed artifact provided: %s", artifact)
 		}
 
-		// TODO: Allow passing Jib config via CLI
-		pairs = append(pairs, builderImagePair{
-			Builder:   docker.Docker{File: parts[0]},
-			ImageName: parts[1],
-		})
+		// Parses JSON in the form of: {"name":"Name of Builder","payload": {...}}
+		// Parse name field first to determine builder type
```

Comment:
I think this comment, the variable name `nameCheck`, and the presence of the `Payload` field is a bit confusing. This is also checking if `Payload` is given. `namePayloadCheck`?

Reply:
The reason it's called `nameCheck` is because we ignore `Payload` until we know what type it is.

Reply:
Hmm, yeah that makes sense, I'll remove the unused fields to hopefully make it more clear.